gnomad_qc/v3/create_release/prepare_vcf_data_release.py

```diff
@@ -812,6 +816,7 @@ def build_parameter_dict(
         freq_entries_to_remove.update(set(COHORTS_WITH_POP_STORED_AS_SUBPOP))
         parameter_dict["freq_entries_to_remove"] = freq_entries_to_remove
         parameter_dict["age_hist_data"] = hl.eval(parameter_dict["ht"].age_distribution)
+        parameter_dict["filtering_model_field"] = "filtering_model"
```

Comment:
just wondering, why is this field named variant_filtering_model only for the hgdp tgp subset?